src/test/java/org/openmrs/contrib/qaframework/automation/AttachmentSteps.java

```diff
@@ -0,0 +1,120 @@
+package org.openmrs.contrib.qaframework.automation;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
+
+import org.openmrs.contrib.qaframework.RunTest;
+import org.openmrs.reference.page.AttachmentsPage;
+import org.openmrs.reference.page.ClinicianFacingPatientDashboardPage;
+import org.openmrs.reference.page.FindPatientPage;
+
+import io.cucumber.java.After;
+import io.cucumber.java.Before;
+import io.cucumber.java.en.And;
+import io.cucumber.java.en.Given;
+import io.cucumber.java.en.Then;
+import io.cucumber.java.en.When;
+
+public class AttachmentSteps extends Steps {
+
+	private AttachmentsPage attachmentsPage;
+
+	@Before(RunTest.HOOK.SELENIUM_DASHBOARD)
+	@Override
+	public void initiatePatientDashboard() {
+		initiateWithLogin();
```

Comment:
you don't need this when you use the @Before(RunTest.HOOK.SELENIUM_DASHBOARD) hook

Reply:
Alright !!